src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimWithDocumentManagementTest.java

```diff
@@ -88,15 +91,40 @@ private void assertSealedClaimIsUploadedIntoDocumentManagementStore(
 
     @Test
     public void shouldLinkSealedCopyOfNonRepresentedClaimAfterUpload() throws Exception {
-        assertSealedClaimIsLinked(SampleClaimData.submittedByClaimant(), AUTHORISATION_TOKEN);
+        assertSealedClaimIsLinked(SampleClaimData.submittedByClaimant(),
+            AUTHORISATION_TOKEN,
+            SEALED_CLAIM,
+            "000MC001-claim-form.pdf");
+    }
+
+    @Test
+    public void shouldLinkDefendantPinLetterAfterUpload() throws Exception {
+        assertSealedClaimIsLinked(SampleClaimData.submittedByClaimant(),
+            AUTHORISATION_TOKEN,
+            DEFENDANT_PIN_LETTER,
+            "000MC001-defendant-pin-letter.pdf");
+    }
+
+    @Test
+    public void shouldLinkClaimIssueReceiptClaimAfterUpload() throws Exception {
+        assertSealedClaimIsLinked(SampleClaimData.submittedByClaimant(),
```

Comment:
assertDocumentIsLinked? because common method and we are using this method to check claim_issue_receipt is also linked